src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { ChangeEvent, FC, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+// eslint-disable-next-line max-len
+const DEFAULT_PREVIEW = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+type Props = {
+  movies: Movie[],
+  onAddMovie: (movies: Movie[]) => void,
+};
+
+export const FindMovie: FC<Props> = ({ movies, onAddMovie }) => {
+  const [query, setQuery] = useState<string>('');
+  const [error, setError] = useState<string | null>(null);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const handleSearch = (event: ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setError(null);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`